### DISCUSSION TITLE
Error "provider" is required while inviting users

### DISCUSSION THREAD

**Discussion by IlyaPetrovM** [directus/directus#16014] [Archive]
Hello.
I am tring to create many users using "Invite users" button. 
I use google Auth provider by default in this project.

**.env file:**
```
...
AUTH_PROVIDERS="google"
AUTH_GOOGLE_DRIVER="openid"
AUTH_GOOGLE_CLIENT_ID="4....5.apps.googleusercontent.com"
AUTH_GOOGLE_CLIENT_SECRET="G......8"
AUTH_GOOGLE_ISSUER_URL="https://accounts.google.com"
AUTH_GOOGLE_IDENTIFIER_KEY="email"
AUTH_GOOGLE_ICON="google"
...
```
my Directus version: 9.9.1
OS: Debian 10

![image](https://user-images.githubusercontent.com/6277090/195994030-f1ad12b4-9525-46fe-b815-84a8369469c9.png)

What happens? How to fix this error?
I don't want to update Directus, but just fix this problem.

**Comment by rijkvanzanten** [comment]
That might be a bug 🤔 Do you have the default login disabled in the env variables? 

Also when you're inviting users through Directus' user directory, do you still want those users to login with Google?

**Reply by IlyaPetrovM** [reply]
Yes, i want new users to login with Google.

How to disable default login in `.env`? I have only those instuctions described above.

**Reply by rijkvanzanten** [reply]
There's a [`AUTH_DISABLE_DEFAULT` env var](https://docs.directus.io/self-hosted/config-options.html#authentication) 👍🏻

**Reply by IlyaPetrovM** [reply]
Does this var work in Directus 9.9.1?

I set `AUTH_DISABLE_DEFAULT=true` but error still appears.

**Reply by rijkvanzanten** [reply]
I don't remember when that was added, should be in the release notes 👍🏻 Staying up to date is very important though, there have been quite a lot of security updates since 9.9

**Reply by IlyaPetrovM** [reply]
OK, I updated Directus to 9.18.1 and still it is not working. 

If I use var `AUTH_DISABLE_DEFAULT=true` the Invite button disappeared completely from Users Directory interface.

**Reply by IlyaPetrovM** [reply]
Ok, I use `AUTH_DISABLE_DEFAULT=true` and `AUTH_<PROVIDER>_ALLOW_PUBLIC_REGISTRATION` 
problem solved.

**Reply by rijkvanzanten** [reply]
> OK, I updated Directus to 9.18.1 and still it is not working.
> 
> If I use var `AUTH_DISABLE_DEFAULT=true` the Invite button disappeared completely from Users Directory interface.

This is expected, as the invite is for local users 🙂 That's also why I asked about the google login earlier. Trying to figure out if public registration was the right tool for the job here 👍🏻
